### PR TITLE
Add python3.8 packages for `aarch64`

### PR DESCRIPTION
**Pull request by galipremsagar**
This PR enables python 3.8 packages for `aarch64` architecture.

Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by galipremsagar** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ source:
     - patches/0001-Fix-software-version-test.patch
 
 build:
-  number: 2
+  number: 3
   script: |
     {{ PYTHON }} -m pip install . -vv
   skip: True  # [win or (aarch64 and py<39)]
```
Comment (jakirkham) [inline recipe/meta.yaml:16]: This appears to disable Python pre-3.9 on ARM. Looks like that was added in PR ( https://github.com/conda-forge/pyorc-feedstock/pull/16 )

@noirello, do you recall why this was needed?

File: .ci_support/linux_aarch64_python3.8.____cpython.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+BUILD:
```
Comment (jakirkham) [inline .ci_support/linux_aarch64_python3.8.____cpython.yaml:1]: We shouldn't be creating this file manually. If conda-smithy is not generating it, there is a reason for that. Please see my other comment for details

Comment (galipremsagar) [reply .ci_support/linux_aarch64_python3.8.____cpython.yaml:1]: Understood, I wasn't aware of it John. The readme also says to change inputs in `conda_build_config.yaml`, but couldn't find it.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -13,7 +13,7 @@ build:
   number: 3
   script: |
     {{ PYTHON }} -m pip install . -vv
-  skip: True  # [win or (aarch64 and py<39)]
+  skip: True  # [win]
```
Comment (jakirkham) [inline recipe/meta.yaml:16]: Also will need to [re-render]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks )

**Comment by galipremsagar** [comment]
@conda-forge-admin, please rerender

File: .ci_support/linux_aarch64_python3.8.____cpython.yaml (hunk 2 above)
Comment (jakirkham) [reply .ci_support/linux_aarch64_python3.8.____cpython.yaml:1]: No worries

We can create `conda_build_config.yaml` in the recipe when it is needed, but that shouldn't be needed in this case

**Comment by jakirkham** [comment]
Looks like we lost CI on the latest commit

@conda-forge-admin, please restart CI

File: .ci_support/linux_aarch64_python3.8.____cpython.yaml (hunk 2 above)
Comment (jakirkham) [reply .ci_support/linux_aarch64_python3.8.____cpython.yaml:1]: Also submitted PR ( https://github.com/conda-forge/conda-smithy/pull/1574 ) to refresh the README in `.ci_support`

**Comment by jakirkham** [comment]
Thanks Prem 😄 Looks like everything passes! 🎉

File: recipe/meta.yaml (hunk 1 above)
Comment (noirello) [reply recipe/meta.yaml]: It had to be that. I wanted to double check, but the logs are no longer available.

Comment (jakirkham) [reply recipe/meta.yaml]: Yeah unfortunately the logs get removed after 30 days

File: .ci_support/linux_aarch64_python3.8.____cpython.yaml (hunk 2 above)
Comment (noirello) [reply .ci_support/linux_aarch64_python3.8.____cpython.yaml:1]: Shouldn't be this file removed, and let the conda-smithy recreate it? Or it doesn't really matter who created it?

Comment (galipremsagar) [reply .ci_support/linux_aarch64_python3.8.____cpython.yaml:1]: I think the rerender command took care of re-modifying/re-creating it.

**Review by noirello — APPROVED** [review]
Thank you, guys. It's much appreciated. :)

**Comment by jakirkham** [comment]
Thanks @noirello! 😄

**Pull request merged**